### PR TITLE
Add TraceBlockchainTime to config files

### PR DESCRIPTION
**Pull request by newhoggy**
`TraceBlockchainTime` is a configuration that was added back in June, but was not mentioned in documentation or in config files found within the project.

This setting must be set to `True` in order to see the "until genesis start time at" when the start time is set to the future.

This PR makes those additions.

Additionally, the `Byron` chairman test now asserts that the text "until genesis start time at" appears in the log file when the start time is in the future.

### CODE REVIEW

File: cardano-node-chairman/src/Chairman/IO/File.hs [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+import           Data.Bool
+import           Data.Functor
+import           Data.String (String)
+import           System.IO (FilePath, IO)
+
+import qualified Data.List as L
+import qualified System.IO as IO
+
+-- | Determin if the given string is found in the given file.
```
Comment (mrBliss) [inline cardano-node-chairman/src/Chairman/IO/File.hs:13]:
```suggestion
-- | Determine if the given string is found in the given file.
```

**Review by mrBliss — COMMENTED** [review]
Thank you for adding this! I have wondered a few times why it was/went missing but never investigated.

File: cardano-node-chairman/src/Chairman/IO/File.hs (hunk 1 above)
Comment (newhoggy) [reply cardano-node-chairman/src/Chairman/IO/File.hs]: Fixed.  Thanks!

File: cardano-node-chairman/test/Test/Cardano/Node/Chairman/Byron.hs [hunk 2]
```diff
@@ -88,7 +95,9 @@ prop_spawnByronCluster = H.propertyOnce . H.workspace "chairman" $ \tempAbsPath
     H.createDirectoryIfMissing $ tempBaseAbsPath <> "/" <> socketDir
 
     H.copyFile (baseConfig <> "/topology-node-" <> si <> ".json") (tempAbsPath <> "/topology-node-" <> si <> ".json")
-    H.copyFile (baseConfig <> "/config-" <> si <> ".yaml") (tempAbsPath <> "/config-" <> si <> ".yaml")
+    H.writeFile (tempAbsPath <> "/config-" <> si <> ".yaml") . L.unlines . fmap rewriteConfiguration . L.lines =<<
```
Comment (Jimbo4350) [inline cardano-node-chairman/test/Test/Cardano/Node/Chairman/Byron.hs:98]: Wouldn't this potentially turn off all logging and only leave on: `TraceBlockchainTime: True`

**Review by Jimbo4350 — COMMENTED** [review]
LGTM but just one comment before I approve

File: cardano-node-chairman/test/Test/Cardano/Node/Chairman/Byron.hs (hunk 2 above)
Comment (newhoggy) [reply cardano-node-chairman/test/Test/Cardano/Node/Chairman/Byron.hs:98]: All the other logging would be as per the original configuration files: `configuration/chairman/defaults/simpleview/config-*.yaml`

The original configuration file has a few trace options turned, and those lines are preserved.

I will use this rewrite mechanism to turn on any additional logging as I need it.

Comment (Jimbo4350) [reply cardano-node-chairman/test/Test/Cardano/Node/Chairman/Byron.hs:98]: Ah I misread the code.

**Review by Jimbo4350 — APPROVED** [review]

**Comment by newhoggy** [comment]
bors merge

**Pull request merged**